docs/ps_modules/latest_news.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import datetime
+from pathlib import Path
+from textwrap import dedent, indent, shorten
+from typing import Iterable, List, Dict, Optional, Tuple
+from docutils import nodes
+from docutils.core import publish_doctree
+
+# ------------------------------ Model -----------------------------------------
+@dataclass(frozen=True)
+class Article:
+    filename: str
+    title: str
+    date: datetime
+    keywords: List[str]
+    description: str
+    priority: int
```

Comment:
use a field with a default value of 10, then it can be created in other ways as well.